Web/starter_project_barbies/components/about/ProjectsList.tsx

```diff
@@ -0,0 +1,72 @@
+import { Project } from '@/types/about'
+import Image from 'next/image'
+import { IconContext } from 'react-icons'
+import { FiGithub } from 'react-icons/fi'
+import { IoMdOpen } from 'react-icons/io'
+import projects from '../../data/about/projects.json'
+
+interface ProjectCardProps {
+  project: Project
+  reverseFlex: boolean
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({ project:{image, name, description}, reverseFlex }) => {
+  return (
+    <div
+      className={`flex flex-wrap justify-around ${
```

Comment:
[very-nit] one alternative way to do this if you want is
```
className={{
  flex,
  flexWrap,
  justifyAround,
  flexRowReverse: reverseFlex
}}
```
(not entirely sure if the class names need to be camelCase or 'kebab-case' as strings)

No need to change now, just wanted to show you an alternative way if you want. this way could be useful when you have class names that will be needed only upon a condition (the ones that should always be there will just be listed as if their condition is true). and it would be cleaner expecially if there are multiple of them